analyzers/rspec/cs/S2221_c#.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title": "\"Exception\" should not be caught when not required by called methods",
+  "title": "\"Exception\" should not be caught",
```

Comment:
review note: @zsolt-kolbay-sonarsource suggested in offline talk that this title can be improved to mention `System.Exception`